features/support/api_helpers.rb

```diff
@@ -169,6 +169,15 @@ def publish_brief(brief_id)
   JSON.parse(response.body)['briefs']
 end
 
+def withdraw_brief(brief_id)
+  puts "ID"
+  puts brief_id
+  path = "/briefs/#{brief_id}/withdraw"
+  response = call_api(:post, path, payload: {
```

Comment:
Would be good to do some error handling on the response like we do in api helper functions above this